### PR TITLE
Show user that subtitle limit was reached

### PR DESCRIPTION
**Pull request by fernandog**
@medariox @Labrys suggestions?

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sickbeard/subtitles.py [hunk 1]
```diff
@@ -507,6 +504,10 @@ def run(self, force=False):  # pylint: disable=unused-argument,too-many-statemen
                         logger.log(u'Downloaded %s subtitles for %s S%02dE%02d'
                                    % (', '.join(new_subtitles), ep_to_sub['show_name'],
                                       ep_to_sub["season"], ep_to_sub["episode"]))
+                else:
+                    logger.log(u"%s S%02dE%02d doesn't have all needed subtitles but search limited was reached"
```
Comment (medariox) [inline sickbeard/subtitles.py]: I think this sentence could be more clear.

Comment (fernandog) [inline sickbeard/subtitles.py]: @medariox 
logger.log(u"%s S%02dE%02d has missed subtitle but search limit was reached" ?

Comment (labrys) [inline sickbeard/subtitles.py]: `Subtitle search limit reached while searching for %s S%02dE%02d` if it's a global limit or `Subtitle search limit reached for %s S%02dE%02d` if it's the limit for that episode.  Also likely a warning would be more appropriate, not an error.

Comment (fernandog) [inline sickbeard/subtitles.py]: Its a limit per episode.
if we improve the logic about the limit we could change message to "has missed subtitle but search limit was reached. Please force subtitle search for this episode"

Comment (medariox) [inline sickbeard/subtitles.py]: I would just use
`
Subtitle search limit reached for %s S%02dE%02d
`
for now.

**Pull request merged**